### PR TITLE
Document jersey client hang workaround

### PR DESCRIPTION
**Pull request by carltonwhitehead**

### CODE REVIEW

File: dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java [hunk 1]
```diff
@@ -108,6 +108,10 @@ public void whenCreateRegistrationItShouldPersist() {
         assertThat(addRegistrationResponse1.getStatus()).isEqualTo(HttpStatus.CREATED_201);
         final String registrationId1 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse1);
 
+        /**
+         * work around Jersey client hanging when client1 reused
```
Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: Either change this comment to say client, or the object reference in the method to be client1 (and all usages of said reference).

Comment (carltonwhitehead) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: I'm going to punt on that until after you figure out if the workaround is truly necessary (like we discussed earlier)

Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: That's fine, I'll give it a try after our call.

Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: It works, do you want to reject this PR, and I will submit a new one?

**Pull request closed without merging**